IsTheMartin/mrtnmrls
# Image container modified

## Summary

The image container was modified to include caption inside container, also, the posts and pages that have images were updated with this change.

Image container width was reduced to fit in text length.

## Evidence

|Before|After|
|---|---|
|![image](https://github.com/IsTheMartin/mrtnmrls/assets/38388605/5c0ba45e-ff14-46b7-9486-ae3512e4a192)|![image](https://github.com/IsTheMartin/mrtnmrls/assets/38388605/5f440823-9da7-4998-8b2f-334b943d051d)|

## Concern Information

To test please follow the next steps:

1. Navigate to some page that contains images
2. Observe caption and background image

## Check list

- [ x ] I did add reviewers to this PR
